test/testenv/util.go

```diff
@@ -643,3 +653,20 @@ func CheckStringInSlice(stringSlice []string, compString string) bool {
 	}
 	return false
 }
+
+// GeneratePodNameSlice returns slice of PodNames based on given key and count.
+func GeneratePodNameSlice(formatString string, key string, count int, multisite bool, siteCount int) []string {
```

Comment:
Doesn't look like this is being used..meant for future? Any idea @pdhanoyasplunk or @jambrosiano ?

Reply:
This file is brought in from MC feature branch where this method is used.

Reply:
As discussed, this will cause a conflict in one of the feature branches when the first one gets merged to develop. Will merge this for now, but in the future request is to add common functions to develop and pull them into the feature branches to avoid additional conflict resolution work